docs/product/insights/frontend/network-requests.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Network Requests
+sidebar_order: 10
+description: "Track the performance of your application's HTTP requests and drill into the domains serving them."
```

Comment:
@bcoe  This description is repeated on the '/backend/network-requests' page, is this supposed to be the case?

Reply:
Yes, we actually have the page in both places because the same insights are applicable to both 👍